### DISCUSSION TITLE
Mobile testing

### DISCUSSION THREAD

**Discussion by ghost** [tersiswilvin/Roblox-Legacy-Old-Theme#14] [General]
With some hassle, I managed to get the JS and CSS onto my phone to test out, and I've got to say, it works great!

![Screenshot_20230804_175949_Kiwi Browser.jpg](https://github.com/tersiswilvin/Roblox-2019-Old-Theme/assets/77752310/c91e397d-90b4-4557-9623-8f4745d77cda)

The only problem is that on some pages the scaling is a bit odd for some elements:

**Comment by tersiswilvin** [comment]
I'll look into seeing if I can fix this, although last time I tested it on my phone I didn't see any issues, it might be depending on the resolution of the screen however I'll look into it whenever I can.

**Reply by ghost** [reply]
Probably is the resolution, as when I zoomed out it kind of fixed a few things, but my phone is 1440p so it should be fine?
